.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*/node_modules/conventional-changelog-core/.*
```

Comment:
Yes normally you would ignore `node_modules` completely and rely on the `flow-typed` module to install typings for the project!